src/Munkres.jl

```diff
@@ -136,6 +144,14 @@ function munkres(costMat::AbstractMatrix{T}) where {T<:Real}
         end
     end
 
+    # if need be, postprocess the result matrix to add the removed rows/columns
+    for rowIndex in unmatchableRows
+        Zs = vcat(Zs[1:rowIndex-1, :], zeros(size(Zs, 2)), Zs[rowIndex:end, :])
+    end
+    for colIndex in unmatchableCols
+        Zs = hcat(Zs[:, 1:colIndex-1], zeros(size(Zs, 1)), Zs[:, colIndex:end])
+    end
+
```

Comment:
Should we move these changes into the method below at line 158? In this way, the behavior and performance for `non-missing` input will not be affected by this patch.

`function munkres(costMat::AbstractMatrix{S}) where {T<:Real,S<:Union{Missing,T}}`

Reply:
I'm wondering what is the root cause of the failure on this corner case. Is it due to numeric errors? The current implementation uses `h = typemax(T)`, for float values, it's `inf`. What if we use `prevfloat(typemax(T))` instead? or replace those `missing` values with `prevfloat(typemax(T))`.